docs(tutorial): ss2 instructions (#118)
autowarefoundation/autoware-documentation: docs/tutorials/scenario-simulation/random-test-simulation.md

# Random test simulation

> Note : Before running the test, please follow the [instruction](installation.md) about installation and building of Autoware Universe and Scenario_simulator_v2

Reviewer: ```suggestion
!!! note

    Running the Scenario Simulator requires some additional steps on top of building and installing Autoware, so make sure that [Scenario Simulator installation](installation.md) has been completed first before proceeding.
```

## Running steps

> 1. Move to project directory, where the project is build.

Reviewer: Slight rewording suggestion and typo correction.

```suggestion
1. Move to the workspace directory where Autoware and the Scenario Simulator have been built.
```

2. Source the workspace setup script

   ```bash
   source install/setup.bash
   ```

3. Run

   ```bash
   ros2 launch random_test_runner random_test.launch.py \
   architecture_type:=awf/universe \
   sensor_model:=sample_sensor_kit \
   vehicle_model:=sample_vehicle
   ```

![random_test_runner](images/random_test_runner.png)

> To modify parameters of random testing, please check documentation of ![random_test_runner](https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/random_test_runner/Readme.md)

Reviewer: Slightly rewording and changed the link to go directly to the section about random test runner parameters.

```suggestion
For more information about supported parameters, refer to the ![random_test_runner documentation].(https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/random_test_runner/Readme.md#node-parameters)
```
